tasties_app/views.py

```diff
@@ -1,63 +1,84 @@
-    return render(request, 'tasties_app/recipes.html', context)
+    recipes_list = Recipe.objects.annotate(
+        recipe_rating=Avg("rating__rating")
+    ).order_by("-recipe_rating")
+    categories_list = Category.objects.all()
+    selected_category = request.GET.get("category")
+    if selected_category:
+        if selected_category != "remove_filter":
+            category = selected_category
+            try:
+                category = Category.objects.get(category_name=selected_category)
+            except Exception:
+                pass
+            recipes_list = (
+                Category.get_recipes_by_category(category)
+                .annotate(recipe_rating=Avg("rating__rating"))
+                .order_by("-recipe_rating")
+            )
+    context = {
+        "recipes_list": recipes_list,
+        "categories_list": categories_list,
+    }
+    return render(request, "tasties_app/recipes.html", context)
 
 
 def login_user(request):
     if request.user.is_authenticated:
-        return redirect('/')
+        return redirect("/")
     else:
-        if request.method == 'POST':
-            username = request.POST.get('username')
-            password = request.POST.get('password')
+        if request.method == "POST":
+            username = request.POST.get("username")
+            password = request.POST.get("password")
             user = authenticate(request, username=username, password=password)
             if user is not None:
                 login(request, user)
-                return redirect('/')
+                return redirect("/")
             else:
-                messages.info(request, 'Username OR password is incorrect')
-    return render(request, 'tasties_app/login.html',)
+                messages.info(request, "Username OR password is incorrect")
+    return render(
+        request,
+        "tasties_app/login.html",
+    )
 
 
 def logout_user(request):
     logout(request)
-    return redirect('login')
+    return redirect("login")
 
 
 def register(request):
     if request.user.is_authenticated:
-        return redirect('recipes')
+        return redirect("index")
 
     form = CreateUserForm()
-    if request.method == 'POST':
+    if request.method == "POST":
         form = CreateUserForm(request.POST)
         if form.is_valid():
             form.save()
-            username = form.cleaned_data.get('username')
-            messages.success(request, 'Account was created for ' + username)
-            return redirect('login')
+            username = form.cleaned_data.get("username")
+            messages.success(request, "Account was created for " + username)
+            return redirect("login")
         else:
             messages.error(request, form.error_messages)
 
-    context = {'form': form}
-    return render(request, 'tasties_app/register.html', context)
+    context = {"form": form}
+    return render(request, "tasties_app/register.html", context)
```

Comment:
All of those changes i.e. switching `'` with `"` or the newlines in return render - make it harder to review this PR.
Please try to avoid it next time.

Reply:
@liornoy @ohadrevach This is probably pre-commit's handywork.

Reply:
Also, per PEP8, both ways work but you should be consistent

Reply:
It is the pre-commit's work, i'll give more attention for it next time.